packages/svelte/tests/runtime-runes/samples/store-subscribe-effect-init/main.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+	import { untrack } from 'svelte';
+	import { writable } from 'svelte/store';
+	import { log } from './log';
+
+	const storeCount = writable(0)
+	let count = $state(0);
```

Comment:
Why make this state, only to untrack it? Couldn't we just do `let count = 0`?

Reply:
It doesn't really have any bearing on this test (which I just copied from the repl).